src/com_tjvendors/media/js/ui/common.js

```diff
@@ -15,7 +15,16 @@ export class CommonUI{
 	{
 		var countryId = jQuery("#" + country).val();
 
-		if (typeof countryId === "undefined") {
+		if (countryId == 0) {
+			jQuery("#jform_region").val(0);
```

Comment:
@deepgawade89 Can you make region and city field dependant on country. Currently, it seems bad UX, you are resetting  region and city to 0. there might be a case where I forgot to add country and I will add on edit

Reply:
As per discussion I am resolving this comment and added task in tracker Task #161442